storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -156,7 +156,10 @@ bool xtrabackup_print_param = FALSE;
 bool xtrabackup_export = FALSE;
 bool xtrabackup_apply_log_only = FALSE;
 
+
 longlong xtrabackup_use_memory = 100 * 1024 * 1024L;
+/* default value max memory is 50% of availabe memory */
+longlong xtrabackup_free_memory_per = 50;
```

Comment:
/*  Default free memory percentage used is 50 % of available free memory */

Reply:
xtrabackup_ prefix can be skipped IMHO..

Reply:
xtrabackup_prefix is used for al other variables.